### PR TITLE
New branch Support Symfony 3.1

### PR DESCRIPTION
**Pull request by gonzaloalonsod**

### CODE REVIEW

**Comment by gonzaloalonsod** [comment]
hola
hello

File: composer.json [hunk 1]
```diff
@@ -1,40 +1,47 @@
 {
-    "name": "symfony/framework-standard-edition",
-    "license": "MIT",
+    "name": "vagrant/turno-rest",
+    "license": "proprietary",
```
Comment (ernestre) [inline composer.json]: Leave the default "name" and "license"

**Comment by gonzaloalonsod** [comment]
Default

**Comment by gonzaloalonsod** [comment]
hello

**Pull request merged**